.github/workflows/tests.yaml

```diff
@@ -363,6 +363,40 @@ jobs:
         source: /tmp/artifacts
       if: always()
 
+  scuba-tests:
+    runs-on: ubuntu-latest
+    needs: build
+    env:
+      S3BACKEND: mem
+      SCUBA_HOST: localhost
+      SCUBA_PORT: 8100
+      SCUBA_HEALTHCHECK_FREQUENCY: 100
+      CLOUDSERVER_IMAGE: ghcr.io/${{ github.repository }}/cloudserver:${{ github.sha }}
```

Comment:
ghcr.io/${{ github.repository }}:${{ github.sha }}

Reply:
I updated the actions.

Reply:
Actually, no, I cannot make this change currently, the PR is based on another one, that is not yet up to date with ghcr migration, so the image in the CI still is the one you commented, with /cloudserver

Reply:
Done after rebasing.